src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -129,9 +129,6 @@ async void InitializeAsync()
             //Initialize the CoreWebView2 component otherwise we can't navigate to a web page
             await documentationBrowser.EnsureCoreWebView2Async();
 
-            // Context menu disabled
-            documentationBrowser.CoreWebView2.Settings.AreDefaultContextMenusEnabled = false;
```

Comment:
I can see that this enables the context menu but by some reason the copy/paste options are not available but anyway I think that will be part of another Jira task.

Reply:
Yes, this at least will enable context menu level copy